skin.json

```diff
@@ -196,7 +196,7 @@
   "icons": {
     "play": {"fontFamilyName": "alice", "fontString": "\u0068", "fontStyleClass": "icon icon-play"},
     "pause": {"fontFamilyName": "alice", "fontString": "\u0067", "fontStyleClass": "icon icon-pause"},
-    "volume": {"fontFamilyName": "alice", "fontString": "\u0062", "fontStyleClass": "icon icon-volume-desktop"},
+    "volume": {"fontFamilyName": "alice", "fontString": "\u0061", "fontStyleClass": "icon icon-volume-desktop"},
```

Comment:
61 is the 'volume-desktop' icon with one sound line, 62 is the 'volume-mobile' icon with 2 sound lines. We were using 61 in the CSS and just had it misrepresented in the skin.json